sample-test/src/main/resources/projects/e2e-test-android.yaml

```
@@ -66,4 +66,12 @@ settings:
       }
-    }
\ No newline at end of file
+    }
+fixedScenarios:
+- id: "3c441e6f-6178-4b4f-b2a2-fbdbf8fa4266"
+  title: "launch setting by maestro"
+  description: "launch setting by maestro"
+  yamlText: |-
+    appId: "com.android.settings"
+    ----
+    - launchApp
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)